tservice/Cargo.toml

```diff
@@ -8,14 +8,19 @@ license = "Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/mobilecoinfoundation/sgx"
 description = "Rust wrappers for the `sgx_tservice` library."
-
 categories = ["api-bindings", "hardware-support"]
 keywords = ["sgx"]
 
+[lib]
+# test false due to needing trts, and thus an enclave to fully link
+test = false
+doctest = false
```

Comment:
Not happy about this, but with the pull in of trts, this means the tests can no longer work outside of an enclave.  I tried playing around with [mockall](https://docs.rs/mockall/latest/mockall/) but it felt a bit heavy handed outside of the `mod test`.